README.md

```diff
@@ -2,7 +2,15 @@
 # libtock-rs
 Rust userland library for Tock (WIP)
 
-Tested with tock a3b36d5872315ff05ef5ad34ed9453b0789218ce.
+Tested with a tock [master from 2019-10-09](https://github.com/tock/tock/commit/6519a10275fc9b673cd78f612701af282e937d29)
```

Comment:
Would it make sense to pin a release version of Tock instead of a random commit, e.g. the freshly released [Tock 1.4](https://github.com/tock/tock/releases/tag/release-1.4)?